src/api/marking/marking.router.ts

```diff
@@ -22,8 +26,9 @@ export const markingRouter = express.Router();
 markingRouter.get(
   "/",
   catchErrors(async (req: Request, res: Response) => {
-    const markings = await getAllMarkings();
-    return res.status(200).json(markings);
+    return res
+      .status(200)
+      .json(array(markingResponseSchema).parse(await getAllMarkings()));
```

Comment:
I would create a variable for line 31. const parsedMarkings = array(....
just makes it more readable